NewHorizons/Utility/NewHorizonBody.cs

```diff
@@ -17,10 +18,43 @@ public NewHorizonsBody(PlanetConfig config, IModBehaviour mod, string relativePa
 
         public PlanetConfig Config;
         public IModBehaviour Mod;
+        public Cache Cache;
         public string RelativePath;
 
         public GameObject Object;
 
+        #region Cache
+        public void LoadCache()
+        {
+            if (RelativePath == null) 
```

Comment:
what if string.isnullorempty instead

Reply:
default value passed in is null so i think this is okay

Reply:
default value of constructor is null so i think this is okay